core/domain/skill_domain.py

```diff
@@ -802,19 +908,21 @@ def serialize(self):
         # version of this object. The line below must then be moved to
         # to_dict().
         skill_dict['version'] = self.version
-
+        # Using type ignore[misc] below because to_disc()
+        # creates a dict with no created_on and last_updated keys.
```

Comment:
I don't understand this.

Reply:
Hi, sorry for the unclear comment. The to_dict() function of the Skill class returns a dictionary with no 'created_on' and 'last_updated' keys so that throws the 'missing keys' as shown below:
![Screenshot 2022-01-28 at 10 34 38 PM](https://user-images.githubusercontent.com/28788192/151590226-5e1b6d58-82fb-48f2-877f-5c7abb8ce313.png)
 I tried various workarounds but couldn't resolve the problem so I decided to use a type ignore. Is there something else I could do to resolve the issue?

Reply:
Sorry for the unclear comment. I have made the changes and resolved the issue instead of using a type ignore.

Reply:
Done

Reply:
Done